src/components/RequesterPaysModal.tsx

```diff
@@ -24,10 +25,14 @@ interface RequesterPaysModalProps {
 }
 
 const RequesterPaysModal: React.FC<RequesterPaysModalProps> = ({ onDismiss, onSuccess }) => {
-  const { workspaces, loading } = useWorkspaces();
+  const { workspaces, loading } = useWorkspaces(['accessLevel', 'canCompute', 'workspace'] as FieldsArg);
```

Comment:
I'm kind of suprised you needed to cast this. It looks like it should work without it.

Reply:
Oh, I probably didn't need to… I was having some other issue. I can remove it, or make it "satisfies" at least, which should enforce the types.

Reply:
I verified that with "satisfies" you get an error immediately in the IDE if a non-string is present in the array. Without out "satisfies", you don't know there is an issue until you build. Not a huge deal for such a trivial object, but I'm trying to get better about leveraging the types.